src/Raven.Server/Documents/Indexes/Static/NuGet/ProjectContext.cs

```diff
@@ -0,0 +1,92 @@
+﻿using System;
+using System.Xml.Linq;
+using NuGet.Common;
+using NuGet.Packaging;
+using NuGet.ProjectManagement;
+using Sparrow.Logging;
+
+namespace Raven.Server.Documents.Indexes.Static.NuGet
+{
+    public class ProjectContext : INuGetProjectContext
+    {
+        private static readonly Logger Logger = LoggingSource.Instance.GetLogger<RavenServer>("NuGet");
+
+        public PackageExtractionContext PackageExtractionContext { get; set; }
+
+        public ISourceControlManagerProvider SourceControlManagerProvider => null;
+
+        public ExecutionContext ExecutionContext => null;
+
+        public XDocument OriginalPackagesConfig { get; set; }
+        public NuGetActionType ActionType { get; set; }
+        public Guid OperationId { get; set; }
+
+        public void Log(MessageLevel level, string message, params object[] args)
+        {
+            if (level == MessageLevel.Debug)
```

Comment:
Should we filter that? If they enabled logging, how do we see that?

Reply:
I do not know. Very low level messages. Your call.

Reply:
Leave it like that for now